boards/dactyl_tracer/README.md

```diff
@@ -0,0 +1,28 @@
+# Dactyl Tracer
+
+![TopElev](https://i.imgur.com/ReCBppEh.jpeg)
+
+A split body, concave shaped key well, columnar keyboard, with case halves designed similarly to the [Dactyl CC](/boards/dactyl/README.md#dactyl-cc--ergo-s-1).  
+The 'Q' key is vertically offset, from 'A' key, to allow more comfortable actuation using ring finger.  
```

Comment:
Allthough I'm a big fan of single quotes in code, I argue it should be double quotes in plain text for "Q" and "A". Did you find any established format preference in the docs?

Reply:
Nothing [here](​​http://kmkfw.io/docs/contributing#contributing-documentation) about quote preference for documentation.

Reply:
single quotes changed to double quotes.